prisma/migrations/20230108225544_init_migration/migration.sql

```diff
@@ -0,0 +1,71 @@
+-- CreateTable
+CREATE TABLE "Category" (
+    "id" SERIAL NOT NULL,
+    "name" VARCHAR(30) NOT NULL,
+    "icon" VARCHAR(30) NOT NULL,
+    "color" CHAR(7) NOT NULL,
+
+    CONSTRAINT "Category_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "Currency" (
+    "id" SERIAL NOT NULL,
+    "code" CHAR(3) NOT NULL,
+
+    CONSTRAINT "Currency_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "Marketplace" (
+    "id" SERIAL NOT NULL,
+    "name" TEXT NOT NULL,
+    "address" TEXT NOT NULL,
+
+    CONSTRAINT "Marketplace_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "Receipt" (
+    "id" SERIAL NOT NULL,
+    "date" TIMESTAMP(3) NOT NULL,
+    "price" DOUBLE PRECISION NOT NULL,
+    "currencyId" INTEGER NOT NULL,
+    "marketplaceId" INTEGER NOT NULL,
+
+    CONSTRAINT "Receipt_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "Article" (
+    "id" SERIAL NOT NULL,
+    "name" VARCHAR(64) NOT NULL,
+    "unitPrice" DOUBLE PRECISION NOT NULL,
+    "amount" DOUBLE PRECISION NOT NULL,
+    "categoryId" INTEGER NOT NULL,
+    "receiptId" INTEGER NOT NULL,
+    "currencyId" INTEGER NOT NULL,
+
+    CONSTRAINT "Article_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateIndex
+CREATE UNIQUE INDEX "Currency_code_key" ON "Currency"("code");
+
+-- CreateIndex
+CREATE UNIQUE INDEX "Marketplace_address_key" ON "Marketplace"("address");
```

Comment:
Not sure if this should be unique. 🤔 
There might be multiple marketplaces at the same address.